tensorflow_addons/optimizers/moving_average.py

```diff
@@ -127,11 +127,14 @@ def get_config(self):
         return {**base_config, **config}
 
     def _create_slots(self, var_list):
-        self._optimizer._create_slots(
-            var_list=var_list
-        )  # pylint: disable=protected-access
+        self._optimizer._create_slots(var_list=var_list)  # pylint: disable=protected-access
```

Comment:
we don't use pylint anymore, can you remove the comment `# pylint: xxx`? Thanks!

Reply:
Gotcha. Removed